pkg/runtime/framework/plugins/torch/torch.go

```diff
@@ -140,3 +137,31 @@ func (t *Torch) EnforceMLPolicy(info *runtime.Info, trainJob *trainer.TrainJob)
 
 	return nil
 }
+
+func (t *Torch) Validate(runtimeJobTemplate client.Object, runtimeInfo *runtime.Info, oldObj, newObj *trainer.TrainJob) (admission.Warnings, field.ErrorList) {
+	var allErrs field.ErrorList
```

Comment:
```suggestion
	if runtimeInfo.RuntimePolicy.MLPolicy != nil || runtimeInfo.RuntimePolicy.MLPolicy.Torch != nil || newObj.Spec.Trainer.NumProcPerNode != nil {
		return nil, nil
	}
	var allErrs field.ErrorList
```

I think we can early return at this point.

Reply:
@tenzen-y we should still validate nprocPerNode here as CEL couldn't access runtimeConfig to validate. Here, nprocPerNode will have different validations depending on the type of mlpolicy in runtime.